CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# dbt_hubspot v1.3.0
+
+[PR #186](https://github.com/fivetran/dbt_hubspot/pull/186) and [PR #187](https://github.com/fivetran/dbt_hubspot/pull/187) include the following updates:
+
+## Schema/Data Change
+**8 total changes • 6 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ------------- | ----------- | ----| --- | ----- |
+| `stg_hubspot__owner_team`<br>`stg_hubspot__form`<br>`stg_hubspot__role`<br>`stg_hubspot__team`<br>`stg_hubspot__team_user`<br>`stg_hubspot__users` | New column | | `is_deleted` (aliased from `_fivetran_deleted`) | Boolean indicating whether a record has been deleted in Hubspot and/or inferred deleted in Hubspot by Fivetran |
+| All models | New column | | `source_relation` | Identifies the source connection when using multiple hubspot connections |
+| `hubspot__company_history` | Updated surrogate key | `id` = `company_id` + `field_name` + `valid_from` | `id` = `source_relation` + `company_id` + `field_name` + `valid_from` |  |
+| `hubspot__contact_history` | Updated surrogate key | `id` = `contact_id` + `field_name` + `valid_from` | `id` = `source_relation` + `contact_id` + `field_name` + `valid_from` |  |
+| `hubspot__deal_history` | Updated surrogate key | `id` = `deal_id` + `field_name` + `valid_from` | `id` = `source_relation` + `deal_id` + `field_name` + `valid_from` |  |
+| `hubspot__daily_ticket_history` | Updated surrogate key | `ticket_day_id` = `date_day` + `ticket_id` | `ticket_day_id` = `source_relation` + `date_day` + `ticket_id` |  |
+| `int_hubspot__pivot_daily_ticket_history`<br>`int_hubspot__ticket_calendar_spine` | Updated surrogate key | `id` = `date_day` + `ticket_id` | `id` = `source_relation` + `date_day` + `ticket_id` |  |
+| `int_hubspot__daily_ticket_history` | Updated surrogate key | `id` = `date_day` + `ticket_id` + `field_name` | `id` = `source_relation` + `date_day` + `ticket_id` + `field_name` |  |
+
+## Feature Update
+- **Union Data Functionality**: This release supports running the package on multiple hubspot source connections. See the [README](https://github.com/fivetran/dbt_hubspot/tree/main?tab=readme-ov-file#step-3-define-database-and-schema-variables) for details on how to leverage this feature.
+
+## Tests Update
+- Removes uniqueness tests. The new unioning feature requires combination-of-column tests to consider the new `source_relation` column in addition to the existing primary key, but this is not supported across dbt versions.
+- These tests will be reintroduced once a version-agnostic solution is available.
+> We have kept uniqueness tests on the surrogate keys listed above.
+
+## Under the Hood
+- In `stg_hubspot__company` and `stg_hubspot__deal`, coalesces `is_<company/deal>_deleted` with `_fivetran_deleted`.
```

Comment:
This actually has the potential to be a breaking change. For example, if I include the following as my passthrough column config, I see a failure due to ambiguous `_fivetran_deleted` columns.
```yml
vars:
    hubspot__company_pass_through_columns: 
      - name: "_fivetran_deleted" 
```

We either need to make sure this is supported or call this out as a breaking change and encourage users to alias the field to ensure there are no ambiguous column failures. Such as:
```yml
vars:
    hubspot__company_pass_through_columns: 
      - name: "_fivetran_deleted" 
        alias: "has_been_deleted"
```